frontend/lindsay-wildlife-treasure-hunt-frontend/src/Components/GameDisplay/ManualEntryDisplay.js

```diff
@@ -46,66 +44,93 @@ export class ManualEntryDisplay extends React.Component {
 
 // Manual Entry Display Components
 
-class ManualEntryField extends React.Component {
+class ManualEntryNumpad extends React.Component {
   constructor(props) {
-    super(props);
+    super(props)
+    this.state = {
+      displayValue: "",
+      displayIsFull: false
+    }
+  }
 
-    this.formPreventDefault = this.formPreventDefault.bind(this);
-    this.onClickPreventDefault = this.onClickPreventDefault.bind(this);
+  digitPressed(digit) {
+    if(!this.state.displayIsFull) {
+      this.setState({
+        displayValue: this.state.displayValue + digit
+      })
+      if(this.state.displayValue.length == 1) {
+        this.setState({
+          displayIsFull: true
+        })
+      }
+    }
   }
 
-  formPreventDefault(e) {
-    e.preventDefault();
-    this.props.onClick();
+  backspace() {
+    this.setState({
+      displayValue: this.state.displayValue.substring(0, this.state.displayValue.length - 1),
+      displayIsFull: false
+    })
   }
 
-  onClickPreventDefault(e) {
-    e.preventDefault();
-    this.props.onClick();
+  handleClick(e) {
+    if(e['target']['className'].includes("num-btn")) {
+      this.digitPressed(e['target']['id'])
+    }
+    else if(e['target']['className'].includes("btn-del")) {
+      this.backspace()
+    }
+    else if(e['target']['className'].includes("btn-submit")) {
+      if(this.state.displayValue.length > 0) {
+        this.props.submitManualNumpad(this.state.displayValue)
+        this.setState({
+          displayValue: "",
+          displayIsFull: false
+        })
+        window.scrollTo(0,150)
+      }
+    }
   }
 
   render() {
     return(
-      <div className="manual-entry-form">
-        <form
-          onSubmit={this.formPreventDefault}
-        >
-          <input 
-            type="text"
-            id="manual-entry-text-field"
-          >
-          </input>
-          <input
-            className="game-button manual-entry-submit"
-            type="submit"
-            value="Submit"
-            onSubmit={this.onClickPreventDefault}
-          >
-          </input>
-        </form>
+      <div className="numpad-body">
+        <div id="display" className="display">{this.state.displayValue}</div>
+        <button id="btn-del" className="btn-del man-btn" onClick={(e)=> this.handleClick(e)}>&#8592;</button>
```

Comment:
Not a blocker for this PR, but an enhancement to this current implementation would be dynamically creating the manual entry pad buttons based on the artifact DB, instead of making it static.